flox/core.py

```diff
@@ -124,7 +136,9 @@ def _get_optimal_chunks_for_groups(chunks, labels):
 
 
 @memoize
-def find_group_cohorts(labels, chunks, merge=True, method="cohorts"):
+def find_group_cohorts(
+    labels, chunks, merge=True, method: T_Method = "cohorts"
+):  # TODO: reduced method?
```

Comment:
what does this comment mean?

Reply:
method can be `["split-reduce", "cohorts"]` according to the docs. I wonder if it's necessary to use a narrower Literal or just go with this one?

Reply:
Lets use a narrower literal inline.